### ISSUE TITLE
Cyclomatic complexity incorrect due to multi-line comments

### ISSUE THREAD

**Issue by ChrisH07** [codecadwallader/codemaid#190]
When calculating cyclomatic complexity for functions with multi-line comments, the regex for removing multi-line comments is to greedy.  In the function CalculateComplexity the following line:
````C#
functionText = Regex.Replace(functionText, @"/\*.*\*/", String.Empty, RegexOptions.Singleline);
````
should be this instead:
````C#
functionText = Regex.Replace(functionText, @"/\*.*?\*/", String.Empty, RegexOptions.Singleline);
````
The ? prevents .* from matching */ in the string.  The following example will yield a complexity of 1 with the existing implementation and 2 with the change above:
```C
void func()
{
  /*
  Some comment
  */
  if (true)
  {
    return;
  }
  /*
  Another comment
  */
  return;
}
```

**Comment by codecadwallader** [comment]
Thanks for reporting the issue, and the proposed fix.  Would you like to submit that as a pull request so you're listed as a contributor?

**Comment by ChrisH07** [comment]
I'm new to GitHub so if you don't mind waiting I'll give it a shot.

**Comment by codecadwallader** [comment]
Not at all, and would be happy for you to get the experience.  Any questions please just let me know. :)

**Comment by codecadwallader** [comment]
Resolved by #194
